content/05.use-cases-c.md

```diff
@@ -37,6 +37,21 @@ In (Figure @fig:github-diagram), we highlight several elements of good repositor
 GitHub also allows users to describe and store information about software and code dependencies, ensuring that code can run using the same version of software as when it was initially developed.
 This can be achieved through the access of GitHub repositories containing the necessary releases of such software, or through the use of automated Github Actions with specific software versions.
 
+### Automation
+
+<!--*Contributors to this section: PHPB -->
+
+Automation has a strong potential to expand the scale and pace of research in ecology and evolution [see @doi:10.1126/science.abi4692].
+Automation frameworks can be applied from the collection, analysis, scrutiny, to the integration and deployment of data and code in EEB research.
```

Comment:
Can we make these examples more concrete? I think it might be hard to imagine that you could use automation if you don't have some specific (published) examples.  In this case I think very specific examples could be a good "hook".  Just an idea:

>Automation can be used to streamline all stages of the scientific process.  For example, automated data validation during the data collection phase (CITATION), automated modeling or forecasting as data are updated and analyzed (CITATION), automated archiving of versioned data and code (CITATION), and automated deployment of rendered manuscripts (used in this manuscript).